src/main/java/luffy/Luffy.java

```diff
@@ -0,0 +1,45 @@
+package luffy;
+
+import java.io.FileNotFoundException;
+
+/**
+ * Represents a chatbot that has various commands
+ * for users to note down tasks and check them off
+ */
+public class Luffy {
```

Comment:
Peak character, therefore a good choice of bot name! :)

Reply:
Hahaha, peak anime!